fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/AggregatedMaterializedViewRewriter.java

```diff
@@ -279,10 +280,17 @@ private boolean isRollupAggregate(List<ScalarOperator> mvGroupingKeys, List<Scal
 
         Set<ScalarOperator> equalsPredicateColumns = new HashSet<>();
         for (ScalarOperator operator : Utils.extractConjuncts(queryRangePredicate)) {
-            if (operator instanceof BinaryPredicateOperator &&
-                    ((BinaryPredicateOperator) operator).getBinaryType().isEqual() &&
-                    operator.getChild(0).isColumnRef() && operator.getChild(1).isConstantRef()) {
-                equalsPredicateColumns.add(operator.getChild(0));
+            if (operator instanceof BinaryPredicateOperator) {
+                BinaryPredicateOperator binaryPredicateOperator = ((BinaryPredicateOperator) operator);
+                if (!binaryPredicateOperator.getBinaryType().isEqual()) {
+                    continue;
+                }
+
+                if (operator.getChild(0).isColumnRef() && operator.getChild(1).isConstantRef()) {
```

Comment:
NormalizePredicateRule can convert `Constant = ColumnRef`  into `ColumnRef = Constant`, no need to consider  `Constant = ColumnRef`  here

Reply:
Yep. This is a just defend code. no other meanings.